stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 2.8.8
+version: 2.8.9
```

Comment:
You’ll have to bump to major version since it seems to break api. I recommend to also write a migrating from 2.x to 3.0 section in the readme, for the few that use « existingAirflowSecrets »

Reply:
It's a good point, and I was wondering about this. Are we actually breaking anything? I mean the `existingAirflowSecret` never worked anyway because of the scoping issue? I have no issue with changing this version, though, if you still think it's a good idea.

Reply:
if it wasnt working at all, this wont affect anyone. ok for bugfix (so ok for 2.8.9)

Reply:
So maybe we want a major bump now